packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/module-exports/output.js

```diff
@@ -8,15 +8,16 @@ console.log(function () {
 console.log(function () {
   throw new Error("The CommonJS '" + "exports" + "' variable is not available in ES6 modules." + "Consider setting setting sourceType:script or sourceType:unambiguous in your " + "Babel config for this file.");
 }().prop);
-
-exports += function () {
+exports += (function () {
   throw new Error("The CommonJS '" + "exports" + "' variable is not available in ES6 modules." + "Consider setting setting sourceType:script or sourceType:unambiguous in your " + "Babel config for this file.");
-}();
-
-exports = function () {
+}(), function () {
   throw new Error("The CommonJS '" + "exports" + "' variable is not available in ES6 modules." + "Consider setting setting sourceType:script or sourceType:unambiguous in your " + "Babel config for this file.");
-}() + 4;
-
+}());
+exports = (function () {
+  throw new Error("The CommonJS '" + "exports" + "' variable is not available in ES6 modules." + "Consider setting setting sourceType:script or sourceType:unambiguous in your " + "Babel config for this file.");
+}() + 4, function () {
+  throw new Error("The CommonJS '" + "exports" + "' variable is not available in ES6 modules." + "Consider setting setting sourceType:script or sourceType:unambiguous in your " + "Babel config for this file.");
+}());
```

Comment:
I wonder if we can avoid emitting the error twice. I think it's duplicated because `exports += 4` is handled like `exports = exports + 4`, so we emit one error for the read and one for the write.

Reply:
This is super low priority: since the generated code always throw, it's will almost never be included in production code.